plugins/appwrite/cli_token.go

```diff
@@ -0,0 +1,56 @@
+package appwrite
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func CLIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.CLIToken,
+		DocsURL:       sdk.URL("https://appwrite.io/docs/command-line"),
+		ManagementURL: sdk.URL("https://cloud.appwrite.io/"),
+		Fields: []schema.CredentialField{
```

Comment:
If environment variables are supported somehow, could we also check if there's one for the Appwrite endpoint? It is a requirement for the appwrite CLI to communicate with the appwrite server, and it's prompted during my setup (without shell plugins) as well.

If we decide to go ahead with supporting [the CI mode](https://github.com/1Password/shell-plugins/pull/276/), let's make sure to create a field for the endpoint and provision it as a `--endpoint` command line argument.